repo/db/cafe_sessions.go

```diff
@@ -37,11 +38,12 @@ func (c *CafeSessionDB) AddOrUpdate(session *repo.CafeSession) error {
 		return err
 	}
 
+	// TODO: There are props on pb.CafeSession we're not persisting
```

Comment:
Ok here is one thing to make sure is right. There are props on `CafeSession` that aren't being persisted. Do we need to persist them all? If so, do we need to do anything with migration old session data?

Reply:
I think we're okay as is. The repo session was originally meant to store just the pieces we needed in order to use it w/ requests and to easily check beforehand if it's expired to reduce round trips. In fact, we only _really_ need the two tokens themselves (access and refresh) because the other info can be parsed out of those, but that's a pita.